java/src/jmri/jmrix/loconet/cmdstnconfig/CmdStnConfigPane.java

```diff
@@ -106,31 +96,36 @@ public void initComponents(LocoNetSystemConnectionMemo memo) {
                 name = name.substring(0, name.indexOf(' '));
             }
             log.debug("match /{}/", name); // NOI18N
+            if (name.startsWith("Intellibox")) {
+                name = "Default"; // use one (default) properties bundle for both IB-I and IB-II, prevent Resource Not Found
```

Comment:
Would it be better to copy the default properties file to ones for the IB-I and IB-II, instead of building this into the code (in a somewhat hidden way)?

Reply:
That what I initiallly planned for, but it would lead to even more duplicates (in lots of translations) where my first question would be if the Intellibox supports this type of configuration at all. A critical look at the way this cmdstnconfig works shows that the values in just a couple of lines of all the the cmdstn bundles differ; the rest is there to make reserved OpSws disappear. The (as yet unrecognised) Intellibox bundle contains no keys at all, so using that instead produces a blank pane. If that's appropriate for the IB, I would also prefer to use dedicated IB-I and IB-II copies. If we default to the Default bundle for any unrecognised command station, it would be more efficient to listen for the exception.

Reply:
My latest commit sports dedicated IB-I/-II bundles (and disables the tool as it will be in the menu, but won't work on an IB according to the documentation.)